.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+
+name: Linting
+env:
+  BUNDLE_WITHOUT: 'test development default'
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # Runs bundle install and caches gems. See the ruby_test.yml
```

Comment:
I made a comment about this in the other pull request - that `ruby_test.yml` is meaningless in these repos - so feel free to change in all the places where you've copy-pasted, or not!

Reply:
Oh bah... removed.